modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -583,6 +584,19 @@ data "aws_iam_policy_document" "karpenter_controller" {
     actions   = ["iam:PassRole"]
     resources = var.karpenter_controller_node_iam_role_arns
   }
+
+  statement {
```

Comment:
lets make this dynamic if an SQS queue arn is provided

Reply:
@bryantbiggs what do you mean with dynamic?
i've already created a var for `karpenter_sqs_queue_arn` and a local